modules/azure/src/main/java/org/testcontainers/containers/Base64Helper.java

```diff
@@ -0,0 +1,90 @@
+package org.testcontainers.containers;
+
+/**
+ * Taken from jakarta xml bind DatatypeConverter#parseBase64Binary
+ *
+ * @author Onur Kagan Ozcan
```

Comment:
You have to retain the original authors if copied verbatim.

Reply:
Removed :)